SofaKernel/framework/sofa/core/loader/MeshLoader.cpp

```diff
@@ -39,10 +39,15 @@ MeshLoader::MeshLoader() : BaseLoader()
     , d_triangles(initData(&d_triangles,"triangles","Triangles of the mesh loaded"))
     , d_quads(initData(&d_quads,"quads","Quads of the mesh loaded"))
     , d_polygons(initData(&d_polygons,"polygons","Polygons of the mesh loaded"))
+    , highOrderEdgePositions(initData(&highOrderEdgePositions,"highOrderEdgePositions","High order edge points of the mesh loaded"))
```

Comment:
Hi Hervé !
Now we have naming conventions in SOFA:
https://github.com/sofa-framework/sofa/blob/master/GUIDELINES.md
see Rule N5 in Naming section: d_ for data